From: Ben Kohler
Date: 2012-08-13 19:59:01 UTC
Bug: https://bugs.gentoo.org/show_bug.cgi?id=431278

remove distutils fallback which silently fails resulting
in a broken installation

this patch enforces a real failure in case of broken setuptools

--- Babel-0.9.6/setup.py
+++ Babel-0.9.6/setup.py
@@ -16,10 +16,7 @@
 import doctest
 from glob import glob
 import os
-try:
-    from setuptools import setup
-except ImportError:
-    from distutils.core import setup
+from setuptools import setup
 import sys
 
 sys.path.append(os.path.join('doc', 'common'))
